### PR TITLE
Update Morocco holidays

### PR DESCRIPTION
**Pull request by abh31000**
## Proposed change

Update Morocco holidays, add Amazigh new year since Morocco recongnized it as an official holiday this month

## Type of change

- [ ] New country/market holidays support (thank you!)
- [x] Supported country/market holidays update (calendar discrepancy fix, localization)
- [ ] Existing code/documentation/test/process quality improvement (best practice, cleanup, refactoring, optimization)
- [ ] Dependency upgrade (version update)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (a code change causing existing functionality to break)
- [ ] New feature (new python-holidays functionality in general)

## Checklist

- [x] I've followed the [contributing guidelines][contributing-guidelines]
- [x] This PR is filed against `beta` branch of the repository
- [x] This PR doesn't contain any merge conflicts and has clean commit history
- [x] The code style looks good: `make pre-commit`
- [x] All tests pass locally: `make test`, `make tox` (we strongly encourage adding tests to your code)
- [x] The related [documentation][docs] has been added/updated (check off the box for free if no updates is required)

[contributing-guidelines]: https://github.com/dr-prodigy/python-holidays/blob/beta/CONTRIBUTING.rst
[docs]: https://github.com/dr-prodigy/python-holidays/tree/beta/docs/source

### CODE REVIEW

File: holidays/countries/morocco.py [hunk 1]
```diff
@@ -44,6 +44,11 @@ def _populate(self, year):
         # New Year's Day
         self._add_new_years_day("Nouvel an - Premier janvier")
 
+        # Amazigh New Year
```
Comment (KJhellico) [inline holidays/countries/morocco.py:47]: Better to place it after Independence Manifesto Day, to keep the chronological order.

**Review by KJhellico — COMMENTED** [review]
LGTM 👍

**Review by arkid15r — COMMENTED** [review]
Could you add the change related reference links if any?

🇲🇦 LGTM

**Comment by arkid15r** [comment]
It has been merged into beta. Thanks for keeping this up to date!

**Pull request merged**